src/main/java/org/mockito/internal/junit/DefaultStubbingLookupListener.java

```diff
@@ -20,6 +20,7 @@
 class DefaultStubbingLookupListener implements StubbingLookupListener {
 
     Strictness currentStrictness;
+    boolean mismatchesReported;
```

Comment:
Actually having the field public (package visible) made me wonder if anyone could change the state, if the field is private and there's a getter I know instantly this field is supposed to be read elsewhere.